source/application/BioXAS/BioXASSideAppController.cpp

```diff
@@ -45,6 +59,10 @@ bool BioXASSideAppController::setupDataFolder()
 void BioXASSideAppController::initializeBeamline()
 {
 	BioXASSideBeamline::bioXAS();
+	connect( BioXASSideBeamline::bioXAS()->ge32ElementDetector(), SIGNAL(connected(bool)), this, SLOT(onGeDetectorConnectedChanged()) );
+	onGeDetectorConnectedChanged();
```

Comment:
This might cause a crash since `initializationBeamline()` is called in the early stage of `startup()` and `setupUserInterface()` is called after that.

`onGeDtectorConnectedChanged()` will try to update the mw_.

The best place for this call (Ln62, Ln63) is in `setupUserInterfaceImplementation()`.